src/ucp/wireup/wireup_ep.c

```diff
@@ -219,30 +235,78 @@ static ucs_status_t ucp_wireup_ep_pending_add(uct_ep_h uct_ep,
 }
 
 static void
-ucp_wireup_ep_pending_purge(uct_ep_h uct_ep, uct_pending_purge_callback_t cb,
-                            void *arg)
+ucp_wireup_ep_pending_purge_cb(uct_pending_req_t *self, void *arg)
+{
+    ucp_wireup_ep_pending_purge_arg_t *purge_arg =
+        (ucp_wireup_ep_pending_purge_arg_t*)arg;
+    ucp_wireup_ep_t *wireup_ep                   = purge_arg->wireup_ep;
+    uct_ep_h uct_ep                              =
```

Comment:
minor: imo, using single line without alignment by `=` would be more readable in this case